plugin.go

```diff
@@ -51,17 +53,24 @@ type Plugin struct {
 	tracer      *sdktrace.TracerProvider
 	propagators propagation.TextMapPropagator
 	mdw         mdw
+	intcpt      intcpt
 }
 
 func (p *Plugin) Init(cfg Configurer, log Logger) error { //nolint:gocyclo
 	const op = errors.Op("otel_plugin_init")
 
-	// name -> http.otel
-	if !cfg.Has(configurationKey) {
+	if !cfg.Has(httpConfigurationKey) && !cfg.Has(grpcConfigurationKey) {
 		return errors.E(errors.Disabled)
 	}
 
-	err := cfg.UnmarshalKey(configurationKey, &p.cfg)
+	var err error
+	if cfg.Has(httpConfigurationKey) {
+		err = cfg.UnmarshalKey(httpConfigurationKey, &p.cfg)
+	} else {
+		err = cfg.UnmarshalKey(grpcConfigurationKey, &p.cfg)
+
+	}
+
```

Comment:
Here, the `gRPC` configuration doesn't depend on the `gRPC` plugin, since this is the client. But, thus we're going to have a unified config for the `otel`, no need to separate this.